fenwick/src/main/java/org/opencadc/fenwick/Main.java

```diff
@@ -162,6 +159,18 @@ public static void main(final String[] args) {
                                               ? InventoryUtil.loadPlugin(configuredArtifactSelector)
                                               : null;
 
+            final String configuredTrackSiteLocations = Main.readProperty(props, TRACK_SITE_LOCATIONS_CONFIG_KEY,
+                                                                          errorMessage);
+            final boolean trackSiteLocations;
+            if (StringUtil.hasText(configuredTrackSiteLocations)) {
+                trackSiteLocations = StringUtil.hasText(configuredTrackSiteLocations)
+                                     && Boolean.parseBoolean(configuredTrackSiteLocations);
+            } else {
```

Comment:
The comment and assignment here just make defaulting to false more explicit. I asked for "review the config parsing and make every item required". If trackSiteLocations (or anything else!) is not configured, fenwick should fail.

Reply:
Alright.  The logic here was borrowed from Critwall, where each missing configuration is gathered and reported at the end.  Is it preferred to have one pass through and check all configuration, and then pull the values out?  That may be more explicit.

Reply:
I've reworked this.  Please look.